### PR TITLE
docs: Add more cross-references to datamodel.

### PR DESCRIPTION
**Pull request by vpetrovykh**

### CODE REVIEW

File: docs/datamodel/props.rst [hunk 1]
```diff
@@ -34,3 +33,13 @@ source objects can link to the same target via
 out the number of property targets, a :ref:`collection type
 <ref_datamodel_collection_types>` target by itself is considered a
 *single* target.
+
+
+See Also
+--------
+
+Link
```
Comment (1st1) [inline docs/datamodel/props.rst]: Is "link" supposed to be here? Or is it "property"?

File: docs/datamodel/scalars/bytes.rst [hunk 2]
```diff
@@ -28,3 +28,12 @@ Bytes
...
         {false}
+
+
+See Also
+--------
+
+Scalar type
+:ref:`SDL <ref_eql_sdl_scalars>`,
+:ref:`DDL <ref_eql_ddl_scalars>`,
+and :ref:`introspection <ref_eql_introspection_scalar_types>`.
```
Comment (1st1) [inline docs/datamodel/scalars/bytes.rst]: I'd also link relevant operators and functions from each data type page

File: docs/datamodel/props.rst (hunk 1 above)
Comment (vpetrovykh) [reply docs/datamodel/props.rst]: Fixed

File: docs/datamodel/scalars/bytes.rst (hunk 2 above)
Comment (vpetrovykh) [reply docs/datamodel/scalars/bytes.rst]: I've linked the sections from funcops.

**Pull request merged**